app/models/application_submission.rb

```diff
@@ -168,4 +168,8 @@ def self.send_notification_emails!(records)
       JobappsMailer.saved_applications_notification records_by_position, address
     end
   end
+
+  def rejected?
+    !saved_for_later? && reviewed? && !interview.present?
```

Comment:
When an application-submission is saved for later, is it reviewed?

Reply:
Nope.

Reply:
So does this need to say `!saved_for_later` specifically I guess is what I'm going for

Reply:
nope.